libnl-route-3.sym

```diff
@@ -1149,4 +1149,15 @@ global:
 	rtnl_vlan_set_protocol;
 	rtnl_vlan_set_vlan_id;
 	rtnl_vlan_set_vlan_prio;
+  rtnl_mdb_alloc_cache;
+	rtnl_mdb_alloc_cache_flags;
+	rtnl_mdb_get_ifindex;
+	rtnl_mdb_get_family;
+	rtnl_mdb_add_entry;
+	rtnl_mdb_foreach_entry;
+  rtnl_mdb_entry_get_ifindex;
+  rtnl_mdb_entry_get_vid;
+  rtnl_mdb_entry_get_state;
+  rtnl_mdb_entry_get_addr;
+  rtnl_mdb_entry_get_proto;
```

Comment:
please indent the entires following the current style. Also, sort the lines asciibetically.

More important, a linker symbol version must not be modified after there is a stable release. That means, don't add entries to the existing `libnl_3_5` section, instead create a new `libnl_3_6` section.